UIKitDZ/UIKitDZ/ViewController.swift

```diff
@@ -6,11 +6,144 @@
 //
 
 import UIKit
- 
-final class ViewController: UIViewController {
 
+final class ViewController: UIViewController {
```

Comment:
```suggestion
final class ViewController: UIViewController {
```
ViewController переименуй в то он делает